ripple/src/main/webapp/posts.js

```diff
@@ -193,4 +193,13 @@ function createPostElement(userName, userBlobKey, businessName, postBlobKey, cap
   `;
   postElement.innerHTML = content;
   return postElement;
+}
+
+// Unit testing exports set up
+
+function subtract(a, b) {
```

Comment:
is this just a test method that you will be writing unit tests for? (assuming will be deleted in a future PR)

Reply:
yes, the function subtract is just a test method that will be deleted. It was written to confirm that the Jest has been properly set up